### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VorobeyAndrew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,44 @@
-# csad2425ki45vorobeyaa02
+# Details about repository:
+  This is a repository for performing laboratory work on the subject "Computer systems automated design". The main branch is develop. Branches named "feature/develop/<task number>" are created to perform laboratory work.
```
Comment (andrii-s-spitzer-lpnu-ua) [inline README.md:2]: change ki45 to ki405

Comment (VorobeyAndrew) [reply README.md:2]: You mean repository name?

Comment (andrii-s-spitzer-lpnu-ua) [reply README.md:2]: yes

Comment (VorobeyAndrew) [reply README.md:2]: I changed repository name, please check

**Review by andrii-s-spitzer-lpnu-ua — APPROVED** [review]

**Pull request merged**